buildSrc/src/main/java/quilt/internal/tasks/diff/RemapTargetMinecraftJarTask.java

```diff
@@ -0,0 +1,31 @@
+package quilt.internal.tasks.diff;
+
+import java.util.Map;
+
+import quilt.internal.Constants;
+import quilt.internal.tasks.jarmapping.MapJarTask;
+
+public class RemapTargetMinecraftJarTask extends MapJarTask {
+    public static final String TASK_NAME = "remapTargetMinecraftJar";
+
+    public RemapTargetMinecraftJarTask() {
+        super("diff", Constants.PER_VERSION_MAPPINGS_NAME, "named");
+
+        CheckTargetVersionExistsTask checkExists = getTaskByType(CheckTargetVersionExistsTask.class);
+        this.onlyIf(task -> checkExists.getTargetVersion().isPresent());
+        this.dependsOn(DownloadTargetMappingJarTask.TASK_NAME);
+        DownloadTargetMappingJarTask downloadTarget = getTaskByType(DownloadTargetMappingJarTask.class);
+
+        inputJar.convention(() -> fileConstants.unpickedJar);
```

Comment:
Isn't this jar already unpicked with the base mappings/definitions? Shouldn't it be using the target ones?